content/Platform/configs.mdx

```diff
@@ -58,4 +58,14 @@ The history log is available for three entities:
 
 For every history log entry, you will see a *Revert Changes* button inside the diff window.
 
-![image](./img/configs/configs-history-diff.png)
\ No newline at end of file
+![image](./img/configs/configs-history-diff.png)
+
+## Comparing Sets
```

Comment:
The title should match the one in the UI 'Configs Compare'

Reply:
done